app/assets/javascripts/models/ui/project_list.js.coffee

```diff
@@ -89,8 +89,18 @@ ns.ProjectList = do (ko
       @collectionsToDownload().length == 0 && @collectionOnly().length == 0 && @submittedOrders().length == 0 && @submittedServiceOrders().length == 0 && @collectionLinks().length == 0
 
     loginAndDownloadCollection: (collection) =>
-      @project.focus(collection)
-      @configureProject()
+      $('#delayOk').on 'click', =>
+        @project.focus(collection)
+        @configureProject()
+      limit = false
+      limit = collection.tags()['edsc.collection_alerts']['data']['limit'] if collection.tags() && collection.tags()['edsc.collection_alerts']
+      if limit && collection.granuleCount() > limit
+        message = collection.tags()['edsc.collection_alerts']['data']['message']
+        if message && message.length > 0 then $("#delayOptionalMessage").text("Message from data provider: " + message) else $("#delayOptionalMessage").text("")
+        $("#delayWarningModal").modal('show')
+      else
+        @project.focus(collection)
```

Comment:
I think it'd be best to pull our the 'successful' functionality into a method instead of copying and pasting it, in the future if we add work to this process we'd need to duplicate it which can easily be overlooked.